reorganize-2019-01.md

```diff
@@ -0,0 +1,86 @@
+# Reorganization for rstudio::conf 2019-01
+
+The outline below is for day 1 only.
+
+## 01) Two Ways to Teach
+
+-   Introductions
+-   Comparison with movie/play (add comedy improv)
```

Comment:
Can you expand on this? I'm not sure what comedy improv refers to - and frankly, I'm a bit scared I might have to do improv 😱

Reply:
I'm inferring this comparison from the graphic in @garrettgman's slides :-)